custom/icds_reports/migrations/sql_templates/create_datasource_views.sql

```diff
@@ -512,36 +512,33 @@ CREATE VIEW child_health_monthly_view AS
         "child_list".block_id,
```

Comment:
Can you move this view to its own file? It'd be nice to split this up, so that each view doesn't need to be recreated when only once changes

Reply:
@emord I moved also the create table clause because is related with the view definition